packages/sanity/src/core/releases/store/createReleaseStore.ts

```diff
@@ -43,21 +44,56 @@ const QUERY_PROJECTION = `{
 // Newest releases first
 const QUERY_SORT_ORDER = `order(${SORT_FIELD} ${SORT_ORDER})`
 
-const QUERY = `*[${QUERY_FILTERS.join(' && ')}] ${QUERY_PROJECTION} | ${QUERY_SORT_ORDER}`
-
-const LISTEN_OPTIONS: ListenOptions = {
-  events: ['welcome', 'mutation', 'reconnect'],
-  includeResult: true,
-  visibility: 'query',
-  tag: 'releases.listen',
-}
+const QUERY = `*[${QUERY_FILTER}] ${QUERY_PROJECTION} | ${QUERY_SORT_ORDER}`
 
 const INITIAL_STATE: ReleasesReducerState = {
   releases: new Map(),
   state: 'loaded' as const,
   releaseStack: [],
 }
 
+const RELEASE_METADATA_TMP_DOC_PATH = 'system-tmp-releases'
+// todo: remove this after first tagged release
+function migrateWith(client: SanityClient) {
+  return client.observable.fetch(`*[_id in path('${RELEASE_METADATA_TMP_DOC_PATH}.**')]`).pipe(
```

Comment:
this migrates from old system-tmp-releases docments in the background (for projects that may have created them during testing).